docker/intel-dgpu-driver.Dockerfile

```diff
@@ -6,8 +6,8 @@ FROM ${DTK_AUTO} as builder
 
 WORKDIR /build/
 
-# Building cse(MEI) driver
-RUN git clone -b 23WW06.5_555_MAIN --single-branch https://github.com/intel-gpu/intel-gpu-cse-backports.git && cd intel-gpu-cse-backports && export OS_TYPE=rhel_8 && export OS_VERSION="8.6" && make modules && make modules_install
+# Building cse(MEI) driver. We are disabling this for now as its not currently used.
```

Comment:
I would say it is instead of its. The other option is to use the contraction it's. More details here: https://www.grammarbook.com/blog/pronouns/1-grammar-error/